govcd/orgs_test.go

```diff
@@ -0,0 +1,28 @@
+//go:build org || functional || ALL
+
+/*
+ * Copyright 2024 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"github.com/kr/pretty"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_GetOrgs(check *C) {
+
```

Comment:
Should we skip this test on VCD < 10.4.2 given that API endpoint requires API `37.2` https://github.com/vmware/go-vcloud-director/pull/669/files#diff-d48dd6aa67017584bb6e37e8e459a46748e9f2660ff686da9a84b3bb18449792R146

Reply:
Yes. And a good thing you noticed, because it actually works with 37.0, so I also amended the openapi endpoint